openshift-with-appstudio-test/e2e/basictests.go

```diff
@@ -181,6 +181,27 @@ func runAbTests(path string, testSet string, pipeline string, ta *testArgs) {
 		}
 	})
 
+	mavenRepoDetails := getMavenRepoDetails(ta)
+
+	ta.t.Run("Maven repo contains artifacts for dependency builds", func(t *testing.T) {
```

Comment:
This test is running (and failing) under the e2e test which is why you aren't getting a clean run.

Reply:
Interesting... that e2e test appears to not use minikube, hence why it fails when I attempt to tunnel from the maven repo service using minikube. May be tricky to figure out how to do this generically. I have no knowledge of how this e2e test works so this should be fun.

Reply:
I was thinking initially exclude that test under e2e? i.e. make it minikube only?

Reply:
Yes, I had considered that too. One very lazy way would be to check for the presence of the minikube binary.

Reply:
Reading a bit further it looks like initially golang build constraints are being used which unfortunately are per file based (see the Makefile). Where is junit when we need it! I don't know if its feasible to move your test into minikube_test.go (but is there ordering issues then?) Alternatively perhaps in https://github.com/redhat-appstudio/jvm-build-service/blob/main/Makefile#L43 set a variable e.g. `MINIKUBE=true` and use a `t.Skip` ( https://konradreiche.com/blog/how-to-separate-integration-tests-in-go ) if that variable isn't set for your test?

Reply:
Must have missed this comment before @rnc, my bad! I was thinking of setting a flag to skip the test if I can't get port forwarding to work without using kubectl/minikube, so we were thinking along the same lines. Unsure if moving the test to minikube_test.go is feasible.

Reply:
Port forwarding within e2e env is no longer the issue it seems, but in fact the repo pod not having started. Will look into tomorrow.

Reply:
I think the `MINIKUBE=true` and a `t.skip` is the simplest option IMHO.